modules/7.2.9/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+SOURCES_DIR=/tmp/artifacts/
+
+export JAVA_OPTS="${JAVA_OPTS} -Dorg.wildfly.patching.jar.invalidation=true"
```

Comment:
Why do we have this here? I know it was added at some point in the past, but is it needed?

Reply:
I'm not sure, this line was included because of the [CLOUD-3274](https://issues.redhat.com/browse/CLOUD-3274).
But I was able to build  the image without this line too, and the images size were the same.
Do you think that is better to remove it?

Reply:
Oh, thanks, I'd forgotten about that. No we should leave it as it just removes the invalidated files after patching.